content/applications/sales/point_of_sale/pricing/electronic_labels.rst

```diff
@@ -150,3 +150,27 @@ successfully sent to Pricer`. If there is any issue, the system displays an erro
 .. warning::
    If a request sent to Pricer fails, Odoo still considers that the product has been updated. In
    that case, we recommend forcing the update of all tags.
+
+
+Discounts
+---------
+
+To display a discount on a Pricer Tag, you have to link a pricelist to the :guilabel:`product variant`
+associated with the tag.
+
+#. Go to :guilabel:`Product Variants` tab,
```

Comment:
It's better to use a menuselection when there's multiple clicks to getting where you need to
Also, to stay consistent with the punctuation above, I've changed the "," to "."
```suggestion
#. Go to :menuselection:`Point of Sale --> Products --> Product Variants`.
```